libraries/AP_Airspeed/AP_Airspeed_MS4525.cpp

```diff
@@ -156,16 +156,19 @@ void AP_Airspeed_MS4525::_collect()
       are generated when the bottom port is used as the static
       port on the pitot and top port is used as the dynamic port
      */
-    float diff_press_PSI = -((dp_raw - 0.1f*16383) * (P_max-P_min)/(0.8f*16383) + P_min);
+    float diff_press_PSI  = -((dp_raw - 0.1f*16383) * (P_max-P_min)/(0.8f*16383) + P_min);
```

Comment:
I have no idea what you're talking about. Can you explain how this relates to the patch?